composer.json

```diff
@@ -75,6 +76,7 @@
         "test-coverage": "phpunit --colors=always --coverage-clover clover.xml"
     },
     "conflict": {
-        "zendframework/zend-hydrator": "*"
+        "zendframework/zend-hydrator": "*",
+        "laminas/laminas-servicemanager": "<3.13.0"
```

Comment:
`<3.14.0` maybe?

Reply:
I picked 3.13 because it's the lowest version where dropping interop became ok - in these situations, is it just best practice to pick latest?